src/app/workflow/register-workflow-modal/register-workflow-modal.component.ts

```diff
@@ -237,6 +239,15 @@ export class RegisterWorkflowModalComponent implements OnInit, AfterViewChecked,
     this.descriptorValidationPattern = DescriptorLanguageService.workflowDescriptorTypeEnumToExtendedDescriptorLanguageBean(
       descriptorType
     ).descriptorPathPattern;
+    if (descriptorType === Workflow.DescriptorTypeEnum.CWL) {
```

Comment:
I think a switch statement would be better; if we ever 

1. Turn on strictnullchecks
2. Add a new language

Then the compiler would catch a missing language.